DOCUMENTATION.md

```diff
@@ -289,20 +289,23 @@ You can leave out this argument if your service only has one email reply-to addr
 
 ## Send a file by email
 
-Send files without the need for email attachments.
+To send a file by email, add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file.
 
-This is an invitation-only feature. [Contact the GOV.UK Notify team](https://www.notifications.service.gov.uk/support/ask-question-give-feedback) to enable this function for your service.
+The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file.
 
-To send a file by email, add a placeholder field to the template and then upload a file. The placeholder field will contain a secure link to download the file.
+#### Add contact details to the file download page
```

Comment:
The markdown here should be `###` so that we don't have an `H2` heading immediately followed by an `H4` with no `H3` in between.  https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks